common-pg/src/test/scala/com/socrata/pg/store/PGSecondaryUniverseTestBase.scala

```diff
@@ -59,7 +59,8 @@ trait PGSecondaryUniverseTestBase extends FunSuiteLike with Matchers with Before
     // Setup the data columns
     val cols = SoQLType.typesByName.filterKeys(t => !UnsupportedTypes.contains(t) &&
                                                     t != SoQLID.name && // These types are specially added.
-                                                    t != SoQLVersion.name)
+                                                    t != SoQLVersion.name &&
+                                                    t != SoQLInterval.name)
```

Comment:
It looks like there are two groups of types here -- "UsupportedTypes" (json) which seems to have aspirations as eventually being a dataset column type? and then SoQLID and SoQLVersion which are special for other reasons?

Would it be right to group SoQLInterval with UsupportedTypes instead of the id/version?

Reply:
oh, you did add it to UnsupportedTypes! does it need to be added here as well then?

Reply:
We don't need it.  Removed.